### PR TITLE
WT-7443 Add error message when bulk cursor can't get exclusive access to dhandle

### PR DESCRIPTION
**Pull request by AlexeyAnisimovAU**

### CODE REVIEW

**Review by keitharnoldsmith — APPROVED** [review]
LGTM!

File: test/suite/test_bulk03.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    err_msg = '/bulk-load is only supported on newly created objects/'
+
+    # Test that bulk-load objects cannot be opened by other cursors.
+    def test_bulk_load_busy_cols(self):
+        # Create a table with columns
+        self.session.create(self.tablename, 'key_format=5s,value_format=HQ,' +
+                            'columns=(country,year,population),' +
+                            'colgroups=(year,population)')
+
+        # Cerarte a column group
```
Comment (alisonfel) [inline test/suite/test_bulk03.py:52]: Typo with 'Create' + missing `.`

File: test/suite/test_bulk03.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        self.session.create(self.cgname + ':year', 'columns=(year)')
+        self.session.create(self.cgname + ':population', 'columns=(population)')
+
+        # Cerate a column cursor
```
Comment (alisonfel) [inline test/suite/test_bulk03.py:56]: Ditto (typo + full sentence)

File: test/suite/test_bulk03.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        self.session.open_cursor(self.tablename, None)
+
+        # Create a second column cursor in bulk mode. Don't close the insert cursor, we want EBUSY
+        # and the error message
```
Comment (alisonfel) [inline test/suite/test_bulk03.py:60]: Ditto (missing  `.`)

File: test/suite/test_bulk03.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_bulk03.py
+#       colgroup bulk-cursor test.
```
Comment (alisonfel) [inline test/suite/test_bulk03.py:30]: Nitpick: might be good to expand this comment out a bit with a description of what this file is specifically testing.

File: test/suite/test_bulk03.py [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+# Smoke test bulk-load.
+class test_colgroup_bulk_load(wttest.WiredTigerTestCase):
+    basename = 'test_schema01'
+    tablename = 'table:' + basename
+    cgname = 'colgroup:' + basename
+    err_msg = '/bulk-load is only supported on newly created objects/'
+
+    # Test that bulk-load objects cannot be opened by other cursors.
+    def test_bulk_load_busy_cols(self):
+        # Create a table with columns
```
Comment (alisonfel) [inline test/suite/test_bulk03.py:47]: Sorry if I missed this and forgot to mention. Comments need to be full sentences ending with `.` (period)

**Review by alisonfel — COMMENTED** [review]
Good work! Just some minor points with regards to python comments.

**Review by alisonfel — APPROVED** [review]
LGTM! Awesome work

File: test/suite/test_bulk03.py (hunk 5 above)
Comment (AlexeyAnisimovAU) [reply test/suite/test_bulk03.py]: Thank you @alisonfel ! No worries, you may have mentioned that. I could just forget :)

**Comment by AlexeyAnisimovAU** [comment]
Thank you for the review!

**Pull request merged**